ui/partials/perf/comparectrl.html

```diff
@@ -4,8 +4,9 @@
       Loading all results, please wait a minute...
       <img src="img/dancing_cat.gif" />
     </div>
-    <div id="error" ng-if="!dataLoading && errors.length" ng-repeat="error in errors">
-      <div>{{error}}</div>
+    <div id="error" ng-if="!dataLoading && errors.length">
+      <p ng-repeat="error in errors" class="bg-danger">{{error}}</p>
+      <a href="#/comparechooser?originalProject={{originalProject}}&originalRevision={{originalRevision}}&newProject={{newProject}}&newRevision={{newRevision}}">Modify choice</a>
```

Comment:
Fwiw we used font awesome for alert icons in Treeherder, they seemed to play nicely with notifications.
https://github.com/mozilla/treeherder-ui-deprecated/pull/513
```
fa-ban (error)
fa-warning
fa-info-circle (info)
fa-check (success)
```